src/js/components/Navigation/HeaderBar.jsx

```diff
@@ -297,71 +297,69 @@ export default class HeaderBar extends Component {
 
     return (
       <header className="page-header">
-        <div className="page-header__content">
```

Comment:
This diff looks horrible but the only change is that the `page-header__content` div was removed and indentation was updated for nested content.